lib/github_usage_tracker.rb

```diff
@@ -0,0 +1,21 @@
+class GithubUsageTracker
+  def self.record_datapoint
+    new.record_datapoint
+  end
+
+  def record_datapoint
+    # TODO Clean this mess up
```

Comment:
This will be cleaned up with the direction we've discussed, which I'll be doing once I get this little feature wrapped up.